aws/resource_aws_network_acl_rule_test.go

```diff
@@ -632,15 +643,13 @@ resource "aws_network_acl" "test" {
 }
 
 resource "aws_network_acl_rule" "test" {
-  from_port       = -1
```

Comment:
removed these port ranges as the protocol here is `ipv6-icmp` and port range is supported only for ucp/tcp per [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-network-acl-entry.html). if port ranges are passed to the test, the api returns nil for `resp.PortRange`; response output from `findNetworkAclRule`: 
```
  Egress: false,
  IcmpTypeCode: {
    Code: -1,
    Type: -1
  },
  Ipv6CidrBlock: "::/0",
  Protocol: "58",
  RuleAction: "allow",
  RuleNumber: 150
}
```